_pages/welcome-to-TTS/getting-started.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Getting started at TTS
+tags:
+- welcome
+- onboarding
+- getting started
+---
+
+
+Getting started at TTS: Your first 90 days
```

Comment:
```suggestion
Your first 90 days
```

Reply:
Rejecting only because I changed up the heading altogether.